src/paperless/settings.py

```diff
@@ -102,7 +104,7 @@ def __get_boolean(key, default="NO"):
 ]
 
 # We allow CORS from localhost:8080
-CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "localhost:8080").split(","))
+CORS_ORIGIN_WHITELIST = tuple(os.getenv("PAPERLESS_CORS_ALLOWED_HOSTS", "http://localhost:8080").split(","))
```

Comment:
This [only applies for django-cors-headers version 3](https://github.com/adamchainz/django-cors-headers/blob/master/HISTORY.rst#300-2019-05-10), and the Paperless [lockfile still references 2.4.0](https://github.com/the-paperless-project/paperless/blob/master/Pipfile.lock#L126-L133). So unless you have upgraded the package on your installation, this should not fail.

What I'm getting at: we should either update both the plugin and the whitelist, or none. Also: if you decide to update the plugin to v3, can you do that in a separate PR?

Reply:
I stumbled over this as I maintain paperless' FreeBSD port which is rolling release - so the package updated to a newer version there. Will adding http:// here break anything on django-cors-headers 2.4.0? (I can of course also update the plugin to three).

Reply:
I'll back out that change for now and open a new pull request later, as suggested

Reply:
Ah, I didn't realize that the scheme was supported, but simply optional before. That definitely makes changing this a no-brainer, even if you don't update the plugin. A separate PR for this sounds great 👍

Reply:
Actually, I'm not entirely sure if 2.4 can handle schemes, so I'll go with what you suggested once we're done here (separate PR and update plugin to 3)